tests/page-objects/contacts/contacts.wdio.page.js

```diff
@@ -52,6 +51,15 @@ const rightAddPerson = (create_key) => $(`span[test-id="rhs_add_contact"] p[test
 const contactCards = () => $$('.card.children');
 const districtHospitalName = () => $('[name="/data/district_hospital/name"]');
 const childrenCards = () => $$('.right-pane .card.children');
+const newActionContactButton = () => $('.action-container .right-pane .actions .mm-icon .fa-stack');
+const forms = () => $$('/html/body/app-root/div/mm-actionbar/div/div/div/div[2]/div/span/ul/li');
+const formTitle = () => $('#form-title');
+const contactCardTitle = () =>
+  $('/html/body/app-root/div/div[2]/div/ng-component/div[2]/div/' +
+    'contacts-content/div/div/div/div[1]/div[2]/div/div[1]/h3');
+const contactInfoName = () =>
+  $('/html/body/app-root/div/div[2]/div/ng-component/div[2]/div/contacts-content/div/div/div/div[1]/div/div[1]/div');
+const contactInfoSex = () => $('//*[@id="contact_summary"]/div[3]/div/div/p');
```

Comment:
Selectors like this that rely heavily on the structure of the DOM are fragile and prone to breaking with small changes on the page. 

Try to use ids and css classes to find elements by when you can, although be careful of being bound to stylistic attributes since those to may change easily. When no other stable option is available, it is preferred that we use a `test-id` attribute. This means you go into the code for the page and add your own element to that source code. For example, you might have html that looks like this:

```
<div>
  <span>Fred</span>
  <span>Rogers</span>
</div>
```
...the second span is showing the last name, but there's no clean way to select that "last name" field. You would do this:

```
<div>
  <span>Fred</span>
  <span test-id="last-name">Rogers</span>
</div>
```

Now you can make a selector like:
```
$('span[test-id="last-name"]');
```

This also means that if a developer comes along later and changes that html, they'll see the attribute and know it's used in a test so whatever change they make, they can retain the attribute and know the test will keep working as intended.